tools/cross/format.py

```diff
@@ -121,11 +121,10 @@ def run_bazel_tool(
     else:
         tool_path = init_external_dir() / tool_target / "file" / "downloaded"
 
-    if not tool_path.exists():
-        build_target = (
-            f"@{tool_target}//:file" if is_archive else f"@{tool_target}//file"
-        )
-        subprocess.run(["bazel", "build", build_target])
+    build_target = (
+        f"@{tool_target}//:file" if is_archive else f"@{tool_target}//file"
+    )
+    subprocess.run(["bazel", "build", build_target])
```

Comment:
why not simply `bazel run {target} -- {args}`? This is what we do for rustfmt

Reply:
We tried to make this tool faster by not invoking bazel unless we knew we needed to download a tool. But now that we have to do it unconditionally in order to handle upgrades, I guess there's not much point and we can just do bazel run.

I guess we should suppress all the verbose messages it prints though.

Reply:
I'm going to keep the current change set as the simplest fix for the problem. Maybe someone can make the various suggested improvements in a followup.